go.mod

```diff
@@ -107,4 +107,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace github.com/codeready-toolchain/toolchain-common v0.0.0-20230406074039-c486d404e698 => github.com/ranakan19/toolchain-common v0.0.0-20230427041859-801d7f82b8e5
```

Comment:
friendly reminder

Reply:
We don't actually need to add such reminders. We have a special job for that. See `ci-check-gomod / go.mod replacements (pull_request)`. It blocks the PR if a replacement found :)